modules/geocoder_geofield/src/Plugin/Geocoder/Dumper/Geohash.php

```diff
@@ -27,7 +24,7 @@ class Geohash extends Geometry {
    * {@inheritdoc}
    */
   public function dump(Address $address) {
-    return parent::dump($address)->out('geohash');
+    return $this->geophp->load($this->getHandler()->dump($address), 'json')->out('geohash');
```

Comment:
`return (parent::dump($address))->->out('geohash');` ?

Strange that it doesn't work. Am I reading wrong the code?